### ISSUE TITLE
Warn when assigning custom attribute via interpolation

### ISSUE THREAD

**Issue by MeirionHughes** [MeirionHughes/aurelia-template-lint#147]
```html 
<div something="${foo}"></div>
```
should raise an issue.

**Comment by atsu85** [comment]
I guess if tag has also `as-element` attribute containing valid custom element tag name, then it would be ok. For example:
`<div something="${foo}" as-element="my-custom-element"></div>`
